examples/storybook/tsconfig.json

```diff
@@ -10,12 +10,7 @@
     "emitDecoratorMetadata": true,
     "experimentalDecorators": true,
     "target": "es5",
-    "typeRoots": [
-      "node_modules/@types"
-    ],
-    "lib": [
-      "es2018",
-      "dom"
-    ]
+    "typeRoots": ["node_modules/@types"],
```

Comment:
You can drop this line it's already the default value (see [doc](https://www.typescriptlang.org/docs/handbook/tsconfig-json.html#types-typeroots-and-types)).

Reply:
IIUC, the doc says by default 
```
 "typeRoots": ["node_modules/@types", "../node_modules/@types", "../../node_modules/@types" ],
```
is that correct ?

Reply:
Indeed didn't realise we were inside a nested folder.